apps/fabric-website/src/components/App/App.scss

```diff
@@ -80,4 +80,8 @@
   .App {
```

Comment:
So you can't tell in Github's frustrating UI, but this entire block is not wrapped in a :global and none of these styles are actually taking place (including the padding 0 for `.App`

I've moved it under `:global` in https://github.com/OfficeDev/office-ui-fabric-react/pull/4899/commits/f338bdf099cc7be53dba37209c3f507f352efdda

Reply:
Ah, that is so sneaky! Thanks for the catch and fix!!